### PR TITLE
Clown FIX: Adds the clown recorder

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Adds the clown recorder to clown's locker (plays sitcom_laugh on use, 30 SECONDS cd, decreased to 10 SECONDS with CMAG).

sitcom_laugh
Sound downloaded from: https://freesound.org/people/Kinoton/sounds/371562/
sitcom_laugh2
Sound downloaded from: https://freesound.org/people/mrrap4food/sounds/618972/
CC0 license.
## Why It's Good For The Game

That should fix the high number of unfunny clowns we usually get.

Can be a cool item to improve the effect of clown's punchlines.
## Images of changes

https://user-images.githubusercontent.com/77684085/202073940-e82f2452-c802-46fe-aff2-1f68a3fdff7e.mp4

## Testing

- Found the item inside the clown locker and used it with and without cmag.
## Changelog
:cl:
add: Added the clown recorder to clown's locker
/:cl:

### CODE REVIEW

**Comment by hal9000PR** [comment]
Amazing

**Comment by Henri215** [comment]
image
Added item action so it can be used while stored.

File: code/game/objects/items/weapons/clown_items.dm [hunk 1]
```diff
@@ -61,3 +61,33 @@
...
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = SLOT_BELT
+	materials = list(MAT_METAL = 180, MAT_GLASS = 90)
+	force = 2
+	throwforce = 0
+	drop_sound = 'sound/items/handling/taperecorder_drop.ogg'
+	pickup_sound = 'sound/items/handling/taperecorder_pickup.ogg'
+	actions_types = list(/datum/action/item_action/laugh_track)
+	var/cooldown
+	var/laugh_cooldown = 30 SECONDS
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/clown_items.dm:80]: just make this a define

Comment (Henri215) [reply code/game/objects/items/weapons/clown_items.dm]: Like this?

File: code/game/objects/items/weapons/clown_items.dm [hunk 2]
```diff
@@ -61,3 +61,40 @@
...
+	item_state = "analyzer"
+	w_class = WEIGHT_CLASS_SMALL
+	slot_flags = SLOT_BELT
+	materials = list(MAT_METAL = 180, MAT_GLASS = 90)
+	force = 2
+	throwforce = 0
+	drop_sound = 'sound/items/handling/taperecorder_drop.ogg'
+	pickup_sound = 'sound/items/handling/taperecorder_pickup.ogg'
+	actions_types = list(/datum/action/item_action/laugh_track)
+	var/cooldown
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/clown_items.dm:82]:
```suggestion
	var/cooldown = 0
```

**Review by Sirryan2002 — APPROVED** [review]
final request, just needs to give cooldown vars an initial value

**Comment by Henri215** [comment]
Added a second laugh track so it doesnt become annoying and decreased the sitcom_laugh length a little bit.

**Review by farie82 — APPROVED** [review]

**Pull request merged**